src/components/EntryListing/EntryListing.js

```diff
@@ -56,7 +56,6 @@ export default class EntryListing extends React.Component {
           <p>{entry.getIn(['data', inferedFields.descriptionField])}</p>
           : inferedFields.remainingFields && inferedFields.remainingFields.map(f => (
             <p key={f.get('name')} className={styles.cardList}>
-              <span className={styles.cardListLabel}>{f.get('label')}:</span>{' '}
```

Comment:
I agree that this is redundant when viewing all entries of a given type, but it's necessary for the editorial workflow and search, where multiple entry types are present at once. If we're removing this we should add it back in for those views.